### ISSUE TITLE
Include file header copyright

### ISSUE THREAD

**Issue by KamranAsif** [schrodinger/open-source-js-template#2]
Not sure how you want to include this

# File Headers
Files should start with the copyright header, as follows:

```
/**
 * Copyright Schrodinger, LLC
 * All rights reserved.
 */
```

**Comment by wcjordan** [comment]
Yep.  I haven't done anything here yet... there's lots to do before this will be ready like
example source with headers.  Maybe some linting or tool for enforcing.

Also readme / contribution guidelines.  Issue template maybe?  Sample package.json?  Sample build scripts, etc.

**Comment by KamranAsif** [comment]
+1 to issue and PR template
